Snapper/Manager.py

```diff
@@ -0,0 +1,74 @@
+from Snapper.Snapper import Snapper
+from Snapper.AssociationMatrix import AssociationMatrix
+from Relationship.Variable import Variable
+from Relationship.SpearframeRelationship import SpearframeRelationship
+
+
+class Manager:
+
+    def __init__(self):
+        self.sensors = []
+        self.route_map = {}
+        self.variables = []
+        self.snapper = Snapper(self)
+        self.matrix = AssociationMatrix()
+
+    def add_sensor(self, sensor):
```

Comment:
This would require cross-module changes. I am not opposed to it, but feel it is best to actually discuss with @DarrianBlack first since his team is over the sensor module. Other than that, I believe I have addressed everything else you mentioned in your review as of now.